easybuild/scripts/bootstrap_eb.py

```diff
@@ -224,6 +236,8 @@ def stage0(tmpdir):
     from distribute_setup import main as distribute_setup_main
     orig_sys_argv = sys.argv[:]  # make a copy
     sys.argv.append('--prefix=%s' % tmpdir)
+    # url should become: http://raw.githubusercontent.com/hpcugent/easybuild-framework/master/easybuild/scripts/
```

Comment:
please add a clear comment what this implies, i.e. which file will be download (from where) because of this?